### PR TITLE
Add example with ipython

### PR DESCRIPTION
**Pull request by vpicavet**
Add an example with Python. Pardon my French !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,54 @@ import file db_name	Create a new database and import data from the dump file
...
-----
+## Usage from another Python module
 
 If you want to let your users the ability to transparently use a PostgreSQL dabatase without ever noticing it, call `pglite.init_cluster()` at some point (during install)
 and then call `pglite.start_cluster()` at the beginning of your program (and possibly register `pglite.stop_cluster()` at the end with `atexit`). Then you can use
 this "local" cluster with a connection on a local TCP port.
+
+## Example
+
+This example runs on Ubuntu 17.10, but you should be able to reproduce on various environment, including Windows.
```
Comment (elemoine) [inline README.md]: environments

File: README.md [hunk 2]
```diff
@@ -26,9 +26,54 @@ import file db_name	Create a new database and import data from the dump file
...
+First install the environment. You will need an existing PostgreSQL installation.
+Then we install pglite and psycopg2 modules in a virtualenv Python environment.
```
Comment (elemoine) [inline README.md]: virtualenv -> virtual ?

File: README.md [hunk 3]
```diff
@@ -26,9 +26,54 @@ import file db_name	Create a new database and import data from the dump file
...
+In [5]: con = psycopg2.connect('host=localhost port=55432 dbname=postgres')
+In [6]: cur=con.cursor()
+In [7]: cur.execute("""select version()""")
+In [8]: rows = cur.fetchall()
+In [9]: print rows
+[('PostgreSQL 9.6.3 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 6.3.0-12ubuntu2) 6.3.0 20170406, 64-bit',)]
+
+In [10]: pglite.stop_cluster()
+en attente de l'arrêt du serveur.... effectué
+serveur arrêté
```
Comment (elemoine) [inline README.md:78]: Would be better to have the above in English

Comment (mhugo) [reply README.md:78]: Connect with `connect(pglite.cluster_params() + " dbname=postgres")` ?

**Pull request merged**